build-tools/packages/build-cli/src/lib/dangerfile.ts

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-import { Logger } from "@fluidframework/build-tools";
+import { defaultLogger, Logger } from "@fluidframework/build-tools";
```

Comment:
Are these related changes?

Reply:
These are not related to `merge-tool` but logger was not assigned